snuba/cli/optimize.py

```diff
@@ -19,7 +19,7 @@
 @click.option(
     "--storage",
     "storage_name",
-    type=click.Choice(["events", "errors", "transactions"]),
+    type=click.Choice(["events", "errors", "transactions", "transactions_v2"]),
```

Comment:
We do not run optimize on transactions today.

Reply:
I have removed transactions option from optimize job. If we need it in the future we can add it back.